RocketChat/Rocket.Chat
# fix: custom emoji file upload with FileSystem method

## Proposed changes (including videos or screenshots)


The issue was happening due to a lack of await the attempt of delete to finishes 



## Issue(s)

fix https://github.com/RocketChat/Rocket.Chat/issues/29738

## Steps to test or reproduce

1. Go to rocket.chat client 
2. Go to Workspace > Settings > Custom Emoji FileSystem
3. Change to something like: 
![image](https://github.com/RocketChat/Rocket.Chat/assets/20212776/9f573df5-2f09-46a5-a024-910fd3bf28e1)
4. Restart your server 
5. Try to add a new custom emoji In Workspace > Custom Emoji 
6. Notice that no error popups and in the terminal the contributor error appears 


## Further comments
